includes/sdk-troubleshooting-and-debugging/latest-sdk-event-callback.md

```diff
@@ -0,0 +1,3 @@
+#### Event Callback
+
+The event callback will be executed after the event has been sent, for both successful and failed events. You can use this method to monitor the event status and message. You can use this way the event status and the message. Check how to set callback under **[configuration > callback](./#configuration)**.
```

Comment:
> You can use this way the event status and the message. 
a typo or missing a word here?

Reply:
Duplicate with the previous sentence. Removed.